presto-jdbc/src/test/java/io/prestosql/jdbc/TestJdbcPreparedStatement.java

```diff
@@ -943,8 +985,9 @@ public BindAssertion resultsIn(String type, String expectedValueLiteral)
                             .isEqualTo(type);
                     assertThat(rs.getString("bound_as_varchar")).as("bound should cast to VARCHAR the same way as literal " + expectedValueLiteral)
                             .isEqualTo(rs.getString("literal_as_varchar"));
-                    assertThat(rs.getObject("bound")).as("bound value should round trip the same way as literal " + expectedValueLiteral)
-                            .isEqualTo(rs.getObject("literal"));
+                    // TODO (https://github.com/prestosql/presto/issues/6242) ResultSet.getObject sometimes fails
```

Comment:
this does not seem related. Extract?

Reply:
It is related -- i am adding test case with DATE 1970-01-01 and it fails here.